App.Payments/PaymentsModule.cs

```diff
@@ -8,8 +12,38 @@ namespace App.Payments
 {
     public class PaymentsModule : IModule
     {
+        /// <param name="container"></param>
```

Comment:
what is the reason of adding this line ? This comment is useless, because it is self-descriptive